trino/sqlalchemy/dialect.py

```diff
@@ -87,6 +87,10 @@ def create_connect_args(self, url: URL) -> Tuple[Sequence[Any], Mapping[str, Any
             kwargs["http_scheme"] = "https"
             kwargs["auth"] = BasicAuthentication(url.username, url.password)
 
+        if "accessToken" in url.query:
+            kwargs["http_scheme"] = "https"
+            kwargs["auth"] = JWTAuthentication(url.query["accessToken"])
```

Comment:
I basically disagree to guess the authentication type from query parameter. Could you tell me why the below code is insufficient in your use case?
```python
from sqlalchemy import create_engine
from trino.auth import JWTAuthentication

engine = create_engine(
    "trino://<username>@<host>:<port>/<catalog>",
    connect_args={
        "auth": JWTAuthentication("<jwt_token>"),
        "http_scheme": "https",
    }
)
```

Reply:
because there are several apps using the client, where the connection details are provided as a sqlalchemy connection string in a configuration file, command line parameter, etc, and not as a python object

Reply:
I think query key should be snake_case to be the same convention as `http_scheme`.
So, @ms32035 would you mind to change it to `access_token`

Reply:
@dungdm93 changed